### DISCUSSION TITLE
Public repos that use packwerk (and packwerk-extensions) that can help beginners

### DISCUSSION THREAD

**Discussion by alejandroechev** [rubyatscale/packwerk-extensions#14] [Ideas]
I started using packwerk (and packwerk-extensions) recently and I am already a big fan ❤️

One thing I havent found are public repos that use packwerk (and packwerk-extensions) that can serve as a guide for beginners. 

With the different checks that the extensions will now provide there is a wide range of ways that packwerk can be used, so it would be very helpful to have small example repos with different structures of packs that show how these different checks can be used.

**Comment by alexevanczuk** [comment]
Hey @alejandroechev I'm glad you're having a good experience so far. I'm not sure of too many public repos that use these gems yet since I've mostly been hearing of private companies using it. 

This might be a great question for the [Ruby/Rails Modularity Slack Server](https://join.slack.com/t/rubymod/shared_invite/zt-1dgyrxji9-sihGNX43mVh5T6tw18hFaQ)!
